administrator/components/com_admin/sql/updates/mysql/3.7.0-2016-06-14.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE if NOT EXISTS  `#__share_draft`(
+  `id` int(10) NOT NULL AUTO_INCREMENT COMMENT 'Primary Key to #__content',
+  `title` varchar(255) NOT NULL,
+  `created` datetime NOT NULL,
+  `modified` datetime NOT NULL, 
+  `sharelink` varchar(255) NOT NULL, 
+  PRIMARY KEY  (`id`)
+) DEFAULT CHARSET=utf8;
```

Comment:
True. But I believe you will store the token rather than the entire URL
@nikitadhiman ?


On Wed, Jun 15, 2016 at 11:29 PM, RolandD <notifications@github.com> wrote:

> In
> administrator/components/com_admin/sql/updates/mysql/3.7.0-2016-06-14.sql
> <https://github.com/joomla-projects/gsoc16_shareable-draft-content/pull/10#discussion_r67212884>
> :
>
> > @@ -0,0 +1,8 @@
> > +CREATE TABLE if NOT EXISTS  `#__share_draft`(
> > +  `id` int(10) NOT NULL AUTO_INCREMENT COMMENT 'Primary Key to #__content',
> > +  `title` varchar(255) NOT NULL,
> > +  `created` datetime NOT NULL,
> > +  `modified` datetime NOT NULL,
> > +  `sharelink` varchar(255) NOT NULL,
> > +  PRIMARY KEY  (`id`)
> > +) DEFAULT CHARSET=utf8;
>
> @coolbung <https://github.com/coolbung> That will depend what is stored
> in the field I guess.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/joomla-projects/gsoc16_shareable-draft-content/pull/10/files/3b1872bf9b3aaaa6786b3cde1f684b425ee5e504#r67212884>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAaNM77asiNhAH11PH_dmB62g91KPQFtks5qMD2GgaJpZM4I1jrl>
> .
>



-- 
Thanks!
Ashwin Date
techjoomla.com
<http://techjoomla.com/?utm_source=email_signature&utm_medium=email&utm_campaign=ashwin>
| appcarvers.com
<http://appcarvers.com/?utm_source=email_signature&utm_medium=email&utm_campaign=ashwin>
| tekdi.net
<http://tekdi.net/?utm_source=email_signature&utm_medium=email&utm_campaign=ashwin>
@coolbung | @techjoomla | @appcarvers | @tekdinet

If it isn't broken, don't fix it!